exporter/containerimage/export.go

```diff
@@ -81,7 +76,10 @@ func (e *imageExporter) Resolve(ctx context.Context, opt map[string]string) (exp
 			}
 			i.ociTypes = b
 		default:
-			logrus.Warnf("image exporter: unknown option %s", k)
+			if i.meta == nil {
+				i.meta = make(map[string][]byte)
+				i.meta[k] = []byte(v)
+			}
```

Comment:
Should this not be:
```go
			if i.meta == nil {
				i.meta = make(map[string][]byte)
			}
			i.meta[k] = []byte(v)
```
? Same in the OCI one.

Reply:
I decided yes, or at least worthy of a PR: https://github.com/moby/buildkit/pull/558